.github/workflows/prettier.yml

```diff
@@ -0,0 +1,18 @@
+name: Continuous Integration
+'on':
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.2
+        with:
+          prettier_options: '--write **/*.{js,md}'
```

Comment:
If you mean the fact that it's not fixing memberslist.json, I think that's because it's set up to only look at js and md files.

Reply:
Oh, you're literally so right